haiku-apps/projectconceptor/projectconceptor-0.1.0.recipe

```diff
@@ -11,10 +11,9 @@ LICENSE="BSD (3-clause)
 	MIT
 	Open Tracker License
 	Zlib"
-REVISION="1"
-srcGitRev="8192c2e9f0c46d4dc316e20b2d3d6ecfe81110ba"
-SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="d76718776fe85d49a6c6137fff0189abff786c3402d2ec197d811391c016e6f2"
+REVISION="2"
+srcGitRev="4bcf8c73c05213322de347f08a2db6696d97a005"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz" CHECKSUM_SHA256="7b80d51bb6ab1ec2a3f22290b3e35dfcc9f667c652ca5c01eb826d86a21abb52"
```

Comment:
```diff
-SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz" CHECKSUM_SHA256="7b80d51bb6ab1ec2a3f22290b3e35dfcc9f667c652ca5c01eb826d86a21abb52"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="7b80d51bb6ab1ec2a3f22290b3e35dfcc9f667c652ca5c01eb826d86a21abb52"
```
BTW, it also builds on x86 and the app seems to be working fine.
```diff
-ARCHITECTURES="x86_gcc2 !x86_64"
+ARCHITECTURES="x86_gcc2 ?x86 !x86_64"
```

(Well, on x86 it crashes when I quit the app, but this might be caused by my x86 system being quite old.)

Reply:
Actually i fixed (the build) yesterday .. because since graphviz is only x86 it will be the main version anyway... (graphviz was a planned backend / editorplugin for this ... years and years ago ;-D)

Reply:
You lost me :confused:

(FWIW, we have a [recipe for graphviz 2.40.1](https://github.com/haikuports/haikuports/blob/68bd91f476304d1d12989c797eb36dee5d77d209/media-gfx/graphviz/graphviz-2.40.1.recipe) which works on x86 and x86_64.)